### PR TITLE
Setup a global context in liqoctl

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR refactors `liqoctl` to add:

* [x] Common context for subcommands
* [x] Several flags:
	* [x] Timeout
	* [x] Debug
	* [x] Version

# How Has This Been Tested?

- [ ] Unit Testing

### CODE REVIEW

File: cmd/liqoctl/cmd/root.go [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
+)
 
-func init() {
+// NewRootCommand initializes the tree of commands.
+func NewRootCommand(ctx context.Context) *cobra.Command {
+	// rootCmd represents the base command when called without any subcommands.
+	var rootCmd = &cobra.Command{
+		Use:   "liqoctl",
+		Short: common.LiqoctlShortHelp,
+		Long:  common.LiqoctlShortHelp,
```
Comment (giorio94) [inline cmd/liqoctl/cmd/root.go:19]:
```suggestion
		Long:  common.LiqoctlLongHelp,
```

**Review by giorio94 — APPROVED** [review]
/lgtm

**Review by aleoli — APPROVED** [review]

**Pull request merged**